### PR TITLE
fix: maturin build by adding version

### PR DESCRIPTION
**Pull request by benfdking**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ dev = [
     "pytest>=8.0.0",
     "pytest-cov>=6.0.0",
     "ruff>=0.8.0",
```
Comment (nikolaevx) [inline pyproject.toml:12]: Do we really need ruff?

Comment (benfdking) [reply pyproject.toml:12]: Sorry, I don't quite understand the question; we don't need ruff. 

It's just being used as a linter/formatter for the few python files, is there any particular reason to not use it or have it?

Comment (nikolaevx) [reply pyproject.toml:12]: Oh, my mistake, I thought it was in the regular dependencies, not in dev.

Comment (benfdking) [reply pyproject.toml:12]: No worries! Thanks for looking!

**Pull request merged**